ParsimmonTests/LemmatizerTests.swift

```diff
@@ -31,7 +31,7 @@ class LemmatizerTests : XCTestCase {
             "i",
             "hold",
             "in",
-            "my",
+            "i",
```

Comment:
Not sure why the behavior of `Lemmatizer` changed. Maybe `NSLinguisticTagger` had some behavioral changes?